portfolio/src/main/webapp/leave-comment/script.js

```diff
@@ -1,3 +1,35 @@
+function onSignIn(googleUser) {
+  if ($('#sign-out').length != 0) {
+    return;
+  }
+
+  let loggedUser = googleUser.getAuthResponse().id_token;
+  $('<input />').attr('type', 'hidden')
+    .attr('name', 'userTokenId')
+    .attr('value', loggedUser)
+    .appendTo('#leave-comment-form');
+
+  $('#leave-comment')
+      .append($('<a></a>')
+          .attr('id', 'sign-out')
+          .attr('href', '#')
+          .attr('onclick', 'signOut();')
```

Comment:
It's better to use jQuery's `.click()` method for attaching the handler.

Reply:
Done.